### ISSUE TITLE
Open Specification for WireMock configuration files

### ISSUE THREAD

**Issue by oleg-nenashev** [wiremock/ecosystem#4]
### Proposal

There are many implementations of WireMock, and they have different configuration file formats. It would be nice, in collaboration with maintainers of the implementations, to create an open specification within the WireMock organization that would be used as a **recommended** implementation of the interfaces.

Suggested implementation:

- Standalone repository for the specification
- SemVer versioning

### If there are any existing implementations, prototypes, blogs or etc, let us know!

- **WireMock Java** schemas: https://github.com/wiremock/wiremock.org-sources/tree/master/swagger
- WireMock.NET Admin API reference including stubbing: https://github.com/WireMock-Net/WireMock.Net/wiki/Admin-API-Reference 
- Compatibility note for WireMock.NET: https://github.com/WireMock-Net/WireMock.Net/wiki/Compatibility-WireMock.org

**Comment by jamietanna** [comment]
To confirm, would this be OpenAPI for the RESTful API and then JSON Schema for the actual stubbings? This would be great and would make it much easier for interop 👏🏽

**Comment by oleg-nenashev** [comment]
Exactly. Today has a call with @StefH who is a WireMock.Net maintainer, and he is interested too. I will start a channel on WireMock Slack
